modules/script/script-impl/src/test/resources/graaljs/java-script.js

```diff
@@ -0,0 +1,6 @@
+(function testJava() {
+     var GraalJavaObjectType = Java.type('com.enonic.xp.script.impl.GraalJavaObject');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
     var GraalJavaObjectType = Java.type("com.enonic.xp.script.impl.GraalJavaObject");
```